src/pages/index.ts

```diff
@@ -0,0 +1,16 @@
+import  Admin  from "./admin/Admin";
+import  LandingPage  from "./landingPage/LandingPage";
+import  Login  from "./login/Login";
+import  Signup  from "./signup/Signup";
+import  Therapists  from "./therapists/Therapist";
+import TherapistPage from "./therapistProfile/TherapistProfile";
+import LoginAdmin from "./loginAdmin/LoginAdmin";
+import NotFound from "./notFound/NotFound";
```

Comment:
please import pages from index file, either to have page at index.tsx, or import pgae from './'